src/main/scala/is/hail/driver/Count.scala

```diff
@@ -32,7 +29,7 @@ object Count extends Command {
 
     val (nVariants, nCalledOption) = if (options.genotypes) {
       val (nVar, nCalled) = vds.rdd.map { case (v, (va, gs)) =>
-        (1L, gs.count(_.isCalled).toLong)
+        (1L, gs.hardCallIterator.count(_ >= 0).toLong)
```

Comment:
Which implements `Iterator[Int]` which is boxed.  `Iterator[T].count()` calls `Iterator[T].next(): T` which causes the Int to be boxed.